modules/ROOT/nav.adoc

```diff
@@ -17,28 +17,37 @@
  ** xref:upload-raml-task.adoc[Upload an Existing RAML File to Exchange from API Designer]
  ** xref:spec-api-public-exchange.adoc[API Specification for MuleSoft Connectivity Ecosystem]
 * xref:about-designing-a-mule-application.adoc[Flow Designer]
+ ** xref:fd-tour.adoc[Tour of Flow Designer]
  ** xref:salesforce-to-twilio.adoc[Create Your First Mule Application with Flow Designer]
+ ** xref:fd-elements-of-flows.adoc[The Elements of Flows in Mule Applications]
  ** xref:import-template.adoc[Create a Mule Application Quickly from a Template]
- ** xref:get-mapping-recommendations.adoc[Get a Recommendation for Mapping Payloads for a Transformation]
- ** xref:dynamic-expression-field.adoc[Specify a Dynamic Value in a Text Field]
- ** xref:custom-expression-field.adoc[Create a DataWeave Expression in a Text Field]
- ** xref:refresh-delete-configuration-task.adoc[Refresh or Delete a Module Configuration]
+ ** xref:to-create-a-mule-application-project.adoc[Start Creating a Mule Application from an Existing One]
+ ** xref:about-data-types.adoc[Define and Use Data Types in a Flow]
+  *** xref:fd-supported-simple-data-types.adoc[Simple Data Types that are Supported in Flow Designer]
+  *** xref:fd-custom-data-types.adoc[Custom Data Types in Flow Designer]
+ ** xref:fd-integrating-systems.adoc[Integrate a System into a Mule Application]
+ ** xref:dynamic-expression-field.adoc[Fill in Cards with Values that Change at Runtime]
+ ** xref:custom-expression-field.adoc[Create a DataWeave Expression in a Field]
+ ** xref:fd-add-core-component.adoc[Add Logic, Data Transformations, Error Catching, and More to Flows]
+  *** xref:reference-flow-task-design-center.adoc[Reference One Flow from Another with Flowref]
+  *** xref:error-handling-task-design-center.adoc[Handle Errors with Try]
+ ** xref:fd-transform.adoc[Mapping Output Data to Input Data]
+  *** xref:fd-mapping-examples.adoc[Examples of Mapping with the Transform Card]
+  *** xref:fd-tour-transform-card.adoc[Tour of the Transform Card]
+  *** xref:get-mapping-recommendations.adoc[Get a Recommendation for Mapping Payloads for a Transformation]
+ ** xref:fd-store-data-top.adoc[Store and Reuse Data within a Flow]
+  *** xref:to-create-and-populate-a-variable.adoc[Store All or Part of the Output of a Card in a Variable]
+  *** xref:fd-store-reuse-output.adoc[Store the Entire Output of a Card in a Variable]
  ** xref:manage-dependency-versions-design-center.adoc[Update Dependencies in a Mule-Application Project]
```

Comment:
Remove - (dash) in title "Update Dependencies in a Mule Application Project"
Indeed, I would change all terms of _Mule Application_ to _Mule App_ or vice versa in the nav and docs just to show consistency on the terms as stated in the Scrubbing for Localization project.

Reply:
Actually, the hyphen is necessary when two nouns together are being used as a single adjective.